csrc/python_frontend/python_bindings.cpp

```diff
@@ -125,6 +125,19 @@ Tensor broadcast_in_dim_fn(
       broadcast_dims));
   return output;
 }
+
+struct DimInfo {
+  int64_t index;
+  int64_t size;
+  int64_t stride;
+  int64_t stride_order;
+  std::optional<bool> contiguity = std::nullopt;
+
+  bool notBroadcast() {
```

Comment:
Minor point: `isBroadcast` would better match the corresponding `IterDomain` method.

Reply:
Fair. let me change that.